src/ripple/rpc/impl/RPCHelpers.cpp

```diff
@@ -883,14 +883,16 @@ beast::SemanticVersion const firstVersion("1.0.0");
 beast::SemanticVersion const goodVersion("1.0.0");
 beast::SemanticVersion const lastVersion("1.0.0");
 
+unsigned int ApiMaximumSupportedVersion = 1;
```

Comment:
I also think there may be a case for declaring a `constexpr ApiDefaultMaximumSupportedVersion` in RPCHelpers.h and using that value here, instead of using `1`.

Reply:
`apiMaximumSupportedVersion` is changed back to `constexpr`, after adding the config option. Thanks.